src/pages/schedule/index.js

```diff
@@ -24,7 +24,7 @@ export default function Home({ calendar, upcoming, photo }) {
           </Heading>
           <Text>
             { upcoming.calendarReleaseDate
-              ? 'We\'ll have it up on '+moment(upcoming.calendarReleaseDate).format('MMMM DD')
+              ? 'We\'ll have it up on '+ moment(upcoming.calendarReleaseDate).add(1, 'd').format('MMMM DD')
```

Comment:
I feel like this isn't the best solution, we should rather fix the underlying error rather than band-aid it - did you look into that? If you did and aren't sure how to fix, then I'm fine doing this

Reply:
After some further research, the moment constructors will by default put it in local time. I needed to add .utc after moment so that it takes the actual time given instead of compensating for the local time.